input/spec.md

```diff
@@ -45,3 +45,67 @@ Keys are as follows:
 * Affected_manufacturers : list of string-refs
 * Fixed_versions : list of string-refs
 * references : object of reference objects
+
+#### Exploitation details
+
+These are obtained from various sources, and give details of the type of vulnerability:
+
+* Surface:
+  * `local` - this vulnerability can be exploited by a user with physical access to the device
+  * `remote` - this vulnerability does not need physical access to the device
+  * `app` - this vulnerability can be exploited by a malicious application
+  * `webpage` - this vulnerability can be exploited by malicious code on a webpage
+  * `usb` - this vulnerability requires use of USB debugging to exploit
```

Comment:
Do we want to use `usb-debug` instead to distinguish from the case where USB without debugging is required (e.g. a local attack exploiting the USB driver)

Reply:
Probably is a good idea